lib/close_old_pull_requests.rb

```diff
@@ -64,20 +62,20 @@ def initialize(github)
     end
 
     def clean_old_pull_requests
```

Comment:
This method is little too complex for my taste (and the `else` here is always going to be a red flag for me). At the risk of scope creep, is there a simple step you can think of to improve this even a tiny bit, so we can at least inch towards something cleaner over time?